src/server/requirements.txt

```diff
@@ -17,3 +17,6 @@ requests==2.22.0
 six==1.15.0
 urllib3==1.25.11
 Werkzeug==1.0.1
+flask-login
+flask_wtf
```

Comment:
flask-login and flask_wtf are not used. Feel free to remove. Just make sure to test out the server to ensure no lingering python code depends on it